src/cpu_info.cpp

```diff
@@ -150,6 +162,28 @@ bool CpuInfo::connect_apml_get_family_model_step(uint8_t soc_num )
         }
       }//end of retry
 
+      if (soc_num == 0)
+      {
+         //set false -Absent if GPIO value is high -default is true
+	 cpuvalue = getGPIOValue(P0_Present);
+         if (cpuvalue == 1)
+         {
+            set_cpu_bool_value(soc_num, false, DBUS_Present, CPU_INTERFACE);
+	    sd_journal_print(LOG_INFO, "Warning : %d CPU is absent \n", soc_num);
+            return false;
+         }
+      }
+      else if (soc_num == 1)
```

Comment:
do we need else if here? 
it should be just "if"

Reply:
fixed